include/z64scene.h

```diff
@@ -1,6 +1,8 @@
 #ifndef Z64SCENE_H
 #define Z64SCENE_H
 
+#include "z64.h"
```

Comment:
Including `z64.h` here is unfortunate but I guess we'll fix it whenever we reorganize headers